scripts/operator-versions.json

```diff
@@ -0,0 +1 @@
+{"versions": ["0.0.1", "0.0.2"]}
```

Comment:
I believe this is the simplest/easiest way, but if this information is on the openshift catalog (or any place in the web), I think is better to grab it from there

Reply:
alright, I will get that info from there, instead of adding in a json file

Reply:
alright, I could filter with `jq`: `".tags[] | select(.end_ts == null) | .name"` but why there multiple tags?

Reply:
I suspect the multiple tags are in the API to ensure consistency of the images when they are getting deleted. Also probably for the history of what happened with the images in the registry.

Reply:
when what about the "dirty" tags: '0.0.1nowatchers', '0.0.1fix777nowatchers', '0.0.1fix777' ?

Reply:
ok, ignored them

Reply:
I can confirm that:

```diff
@@ -460,7 +460,8 @@ spec:
   provider:
     name: Red Hat
     url: https://redhat.com
-  version: 0.0.2
-  replaces: pelorus-operator.v0.0.1
-  skips:
-    - pelorus-operator.v0.0.1
+  version: 0.0.3
+  replaces: pelorus-operator.v0.0.2
+    skips:
+      - pelorus-operator.v0.0.2
+      - pelorus-operator.v0.0.1
```

Reply:
oh wait, I think I putted wrong indentation